zanata-adapter-xliff/src/main/java/org/zanata/adapter/xliff/XliffWriter.java

```diff
@@ -163,6 +162,21 @@ private static void writeTransUnitContext(IndentingXMLStreamWriter writer,
         }
     }
 
+
+    /**
+     * Used for writing translation file with given translations map
+     *
+     * @param doc
+     * @param targetDoc
+     * @param file
+     * @param locale (use hyphen, not underscore)
+     * @throws IOException
+     */
+    public static void write(final Resource doc, final TranslationsResource targetDoc,
+        final File file, String locale, boolean createSkeletons) throws IOException {
+        writeFile(file, doc, locale, targetDoc, createSkeletons);
```

Comment:
This looks like it just has all the same arguments passed in a different order. Would it be better to remove this entirely and just change `writeFile` to `public`?